apps/portals/src/configurations/elportal/synapseConfigs/computational_tools.ts

```diff
@@ -22,7 +22,7 @@ export const computationalCardConfiguration: CardConfiguration = {
     {
       isMarkdown: false,
       baseURL: 'Explore/Projects/DetailsPage',
-      URLColumnName: 'grantNumber',
+      URLColumnName: 'grant',
```

Comment:
Fix link to go from grant number on computational tools card to project page (where projects are uniquely identified by a set of grants)

![image](https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/49dcb046-7407-49d4-abad-605866469cc9)